tests/periph_rtt_min/main.c

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2020 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief       Test to figure out RTT_MIN_VALUE
+ *
+ * @author      Francisco Molina <francois-xavier.molina@inria.fr>
+ *
+ * @}
+ */
+
+#include <stdint.h>
+#include <stdio.h>
+#include <stdatomic.h>
+
+#include "cpu.h"
+#include "periph_conf.h"
+#include "periph/rtt.h"
+
+#include "xtimer.h"
+
+#define MAX_WAIT_US     (100)
```

Comment:
Why is this needed?

Reply:
I wanted to wait for a minimum amount of time to account for the time between the callback being set and handled.